.release.json

```diff
@@ -33,12 +33,12 @@
         "git config user.name \"${GIT_USER_NAME}\"",
         "git config user.email \"${GIT_USER_EMAIL}\"",
         "git remote set-url origin \"https://${GH_TOKEN}@${GH_REF}\"",
-        "node ./node_modules/.bin/release --bump ${RELEASE_VERSION} ${DEVELOPMENT_VERSION}",
+        "node ./node_modules/twilio-release-tool/release --bump ${RELEASE_VERSION} ${DEVELOPMENT_VERSION}",
         "git add package.json",
         "git rm -r docs lib",
-        "git rm -r src/constants.ts",
-        "git rm -r ios/TwilioVoiceReactNativeConstants.h",
-        "git rm -r android/src/main/java/com/twiliovoicereactnative/CommonConstants.java",
```

Comment:
Not explained in the description of this PR. What does removing `-r` do? Why did it work before and why do this now?

Reply:
The `-r` flag is for `recursive` and since those are files they don't need to be recursively removed, i.e. `-r` is for directories. When I wrote it, I just copied the previous commands but noticed in this PR that the flag is unnecessary (and potentially dangerous since it _could_ remove whole directories if the command were somehow mutated).